Assets/HoloToolkit/Input/Scripts/InteractableViaEvents.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+
+namespace HoloToolkit.Unity
+{
+    /// <summary>
+    /// Use this class for easy access to OnTap, OnGazeEnder, and OnGazeExit events.
+    /// Be sure to override each of these methods with your own implimentation.
+    /// Examples of code usage in each method below in comments.
+    /// </summary>
+    [RequireComponent(typeof(Collider))]
+    public class InteractableViaEvents : MonoBehaviour
```

Comment:
This script is something anyone will have to edit to use.
Why dont we just introduce a concept of InteractibleManager much like Gaze 210 then?